### ISSUE TITLE
Accuracy problem between the onnx-tensorrt and original pytorch

### ISSUE THREAD

**Issue by oscarriddle** [onnx/onnx-tensorrt#370]
Env:
CentOS7, Pytorch 1.4, TensorRT7, CUDA10.2, cudnn7.6, 

Description:
I downloaded the inception v3 pth model from torchvision and export it to onnx through regular routine, and then successfully built the project [https://github.com/NVIDIA/TensorRT](https://github.com/NVIDIA/TensorRT), and managed to run the onnx2trt to convert onnx v3 model to the trt engine with fp32 precision.

Then I compared the result accuracy between the original pytorch model and trt engine on the same inputs. I found the Top1 class usually can be identical but the Top5 classes don't, not to mention the score feature of each class.

Here I provide the argsorted output of the two methods:
```
Image: Pytorch repo caffe2/trt/data/binoculars.jpeg
TRT engine:
...  773 622 626 504 633 563 759 125 372 447]

Pytorch:
...  101 872 622 626 504 563 759 372 125 447]]
```

```
Image: Pytorch repo caffe2/trt/data/tabby_tiger_cat.jpg
TRT engine:
 ... 778 750 753 283 728 761 287 285 282 281]

Pytorch:
... 475 811 728 753 836 837 287 285 282 281]]
```

I'm very confusing whether this accuracy performance is legit? Because I usually focused on the tensorflow-tensorrt, which I reckon is no more than another parser of tensorrt, but it shows very high accuracy in FP32 or FP16 mode, not only the classes are all identical but also the score features only differs at 1E-5 level.

I suppose onnx-tensorrt or somehow pytorch-tensorrt shall be able to retain the accuracy as well as tftrt (at FP32 precision). It shall be possible isn't it? Nevertheless, I haven't seen it by far.

If you would shall some idea on this, it'll be much appreciated.

**Comment by Semihal** [comment]
@oscarriddle , hi!
Which conversion path do you use?
1. Pytorch -> onnx - > tensorrt or... 
2. Pytorch - tensorrt
?

**Comment by oscarriddle** [comment]
Solved

**Comment by codarag** [comment]
> 
> 
> Solved

Hi, 
I have a similar issue, may I know how you solved it?

**Comment by taxuezcy** [comment]
i have the same problem,how you solved it?

**Comment by taxuezcy** [comment]
@oscarriddle

**Comment by fcc315** [comment]
I am having a similar problem: trt_fp32 inference performance (resnet18, top-1 accuracy) is not the same as the original PyTorch.  Trying to figure out where causes this performance downgrade. Does anyone have any clues?

**Comment by Code-Gratefully** [comment]
@codarag @taxuezcy I set export_params=True as suggested earlier and it solved my problem too
